lib/src/widget/nav_bar/utils/config_mixin.dart

```diff
@@ -1,5 +1,10 @@
 import 'package:xayn_design/src/widget/nav_bar/data/nav_bar_config.dart';
+import 'package:xayn_design/xayn_design.dart';
 
 mixin NavBarConfigMixin {
-  NavBarConfig get navBarConfig;
+  /// If value is null - the previous in a tree [NavBarConfig] will be applied
+  /// If value is  not null - config will be shown.
+  /// If need to hide previous config, and show nothing
+  ///   (hide [NavBar] while widget shown) - use [NavBarConfig.hidden]
+  NavBarConfig? get navBarConfig;
```

Comment:
Just a question, why make it nullable and depend on `null` when we can make a default like `NavBarConfig.default()` that when there, we debounce/get the previous in tree?

Reply:
That was my initial goal, to be honest 😄 
But then what is the difference between `.ignorred()` and `null` literally?

Anyway, if that makes more sense to you, sure, I will update it 😉

Reply:
If it's possible to add `ignored`, it'll be much cleaner and readable